src/anemoi/training/diagnostics/callbacks/__init__.py

```diff
@@ -689,6 +689,10 @@ def _plot(
             in_place=False,
         ).numpy()
 
+        if pl_module.output_mask is not None:
```

Comment:
Perhaps these three lines could be in a method `self.apply_output_mask(data)` ?
_ They would not be duplicated.
_ This would allow more flexibility.

To avoid the two "if" below, you could even have a separate object `self.output_masker.apply_to(data)`

Reply:
Thanks for the comment Florian. I have added an apply_output_mask method to fill the values, but not sure how I could avoid the "if"'s.